packs/abomination-vaults-bestiary/bloodsiphon.json

```diff
@@ -137,7 +137,7 @@
                 "category": "defensive",
                 "deathNote": true,
                 "description": {
-                    "value": "<p>20 feet. When the bloodsiphon dies, its body explodes in a cloudy red burst of necrotic dried blood in a 20-foot area.</p>\n<p>Creatures in range must attempt a @Check[reflex|dc:21|traits:damaging-effect] save.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected.</p>\n<p><strong>Success</strong> The creature takes @Damage[2d6[void]] damage.</p>\n<p><strong>Failure</strong> The creature takes @Damage[4d6[void]] damage.</p>\n<p><strong>Critical Failure</strong> The creature takes @Damage[6d6[void]] damage and is @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 2} for 1 hour.</p>"
+                    "value": "<p>When the bloodsiphon dies, its body explodes in a cloudy red burst of necrotic dried blood in a @Template[emanation|distance:20]{20-foot area}.</p>\n<p>Creatures in range must attempt a @Check[reflex|dc:21|traits:damaging-effect] save.</p><hr /><p><strong>Critical Success</strong> The creature is unaffected.</p>\n<p><strong>Success</strong> The creature takes @Damage[2d6[void]] damage.</p>\n<p><strong>Failure</strong> The creature takes @Damage[4d6[void]] damage.</p>\n<p><strong>Critical Failure</strong> The creature takes @Damage[6d6[void]] damage and is @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 2} for 1 hour.</p>"
```

Comment:
Go ahead and remove the aura trait from this. It's clearly not one so either an us mistake or an obvious Paizo mistake.

Reply:
Will do.
<img width="258" alt="Screen Shot 2024-10-12 at 10 00 40 PM" src="https://github.com/user-attachments/assets/adc922bc-bebe-4c98-b6cf-2c4ae1786770">